### PR TITLE
fix: article metadata mobile issues & hide mini-toc section on mobile EXLM-286, EXLM-293

### PR DESCRIPTION
**Pull request by nitin-rachabathuni**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: https://jira.corp.adobe.com/browse/EXLM-286
https://jira.corp.adobe.com/browse/EXLM-293

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.page/docs/blueprints-learn/architecture/customer-journeys/journey-optimizer/journey-optimizer
- After: https://bugfix-exlm-286-article-tag--exlm--adobe-experience-league.hlx.page/docs/blueprints-learn/architecture/customer-journeys/journey-optimizer/journey-optimizer

### CODE REVIEW

File: blocks/article-metadata-createdby/article-metadata-createdby.scss [hunk 1]
```diff
@@ -6,29 +8,40 @@ ul.meta-created-for {
...
+.meta-created-for li.meta-created-for-element {
+  font-size: 12px;
+  border: 1px solid var(--non-spectrurm-article-light-gray);
+  color: var(--non-spectrurm-article-dark-gray);
+  padding: 4px 9px;
+  padding-top: 3px;
+  cursor: default;
+  -webkit-font-smoothing: subpixel-antialiased;
+  border-radius: 4px;
+  line-height: 15.6px;
```
Comment (thomsebastin) [inline blocks/article-metadata-createdby/article-metadata-createdby.scss:20]: Can tis line-height be a variable? I have been seeing this value quite often now.

File: styles/styles.scss [hunk 2]
```diff
@@ -684,8 +696,28 @@ img {
...
+        }
+      }
+
+      &.rail.rail-left {
+        display: none;
+
+        @include breakpoint(desktop) {
+          display: block;
+        }
+      }
```
Comment (thomsebastin) [inline styles/styles.scss:720]: are these changes related to this PR?

**Review by thomsebastin — CHANGES_REQUESTED** [review]

**Comment by thomsebastin** [comment]
Please merge with latest `main` and see if the performance score changes.

File: styles/styles.scss (hunk 2 above)
Comment (nitin-rachabathuni) [reply styles/styles.scss]: Yes Thomas, these are related to EXLM-293

File: styles/styles.scss [hunk 3]
```diff
@@ -285,8 +286,8 @@
   --non-spectrum-grey-updated: #6e6e6e;
   --non-spectrum-link: #485fc7; /* this provides better contrast for a11y */
   --non-spectrum-navy-blue: #1473e6;
-  --non-spectrurm-article-light-gray: #d3d3d3;
-  --non-spectrurm-article-dark-gray: #505050;
+  --non-spectrum-article-light-gray: #d3d3d3;
+  --non-spectrum-article-dark-gray: #505050;
```
[the remark this answers is not in the document]
Comment (nitin-rachabathuni) [reply styles/styles.scss:290]: I have already updated all the references of these variables @thomsebastin 
--non-spectrum-article-light-gray: #d3d3d3;
  --non-spectrum-article-dark-gray: #505050;

**Comment by thomsebastin** [comment]
I think this particular page and other pages where video is loaded seems to have some performance issues as it is blocking the main thread. We may have to look at it.
cc: @ahmed-musallam

**Review by noormd08 — APPROVED** [review]

File: styles/styles.scss [hunk 4]
```diff
@@ -579,14 +580,18 @@ img {
...
+  padding: 30px 2px;
 
-  & > .default-content-wrapper:last-child > p:last-child {
-    margin-bottom: 0;
+  h1 {
+    line-height: var(--spectrum-line-height-xxl);
+
+    @include breakpoint(desktop) {
+      line-height: var(--spectrum-line-height-xxxl);
+    }
```
Comment (noormd08) [inline styles/styles.scss:590]: This can lead to override throughout application.

**Review by noormd08 — APPROVED** [review]

**Pull request merged**